lib/core/string.toit

```diff
@@ -1391,6 +1403,39 @@ class StringSlice_ extends string:
   compute-hash_ -> int:
     #primitive.core.blob-hash-code
 
+class StringByteSlice_ implements io.Data:
+  str_ / String_
+  from_ / int
+  to_   / int
```

Comment:
Either all three slashes should be aligned or none of them.

Reply:
done.